src/ButteredBread.sol

```diff
@@ -0,0 +1,168 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.25;
+
+import {OwnableUpgradeable} from "openzeppelin-contracts-upgradeable/contracts/access/OwnableUpgradeable.sol";
+import {ERC20VotesUpgradeable} from
+    "openzeppelin-contracts-upgradeable/contracts/token/ERC20/extensions/ERC20VotesUpgradeable.sol";
+import {IERC20} from "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
+import {IButteredBread} from "src/interfaces/IButteredBread.sol";
+
+/**
+ * @title Breadchain Buttered Bread
+ * @notice Deposit Butter (LP tokens) to earn scaling rewards
+ * @author Breadchain Collective
+ * @custom:coauthor @RonTuretzky
+ * @custom:coauthor @daopunk
```

Comment:
For YieldDistributor we added user's who provided feedback to coauthors list, so can you please add @bagelface

  🥹 
👉 👈